procedures/MT/rotator/rotator_performance.ipynb

```diff
@@ -0,0 +1,596 @@
+{
```

Comment:
<p>Please clarify the "setup" comment here as well, as you did for the verification notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/96/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Updated in the commit: [3a3d19c](https://github.com/lsst-ts/ts_notebooks/pull/96/commits/3a3d19ca25f47a8afd6d0e2d48a2c4c3cc33224d). Thanks!